operate/cli.py

```diff
@@ -132,6 +137,42 @@ def __init__(
         mm.migrate_wallets(self.wallet_manager)
         mm.migrate_qs_configs()
 
+    def _backup_operate_if_new_version(self) -> None:
+        """Backup .operate directory if this is a new version."""
+        current_version = Version(__version__)
+        backup_required = False
+        version_file = self._path / VERSION_FILE
+        if not version_file.exists():
+            backup_required = True
+            found_version = "0.10.21"  # first version with version file
+        else:
+            found_version = Version(version_file.read_text())
+            if current_version.major > found_version.major or (
+                current_version.major == found_version.major
+                and current_version.minor > found_version.minor
+            ):
+                backup_required = True
+
+        if not backup_required:
+            return
+
+        backup_path = self._path.parent / f"{OPERATE}_v{found_version}_bkp"
+        logger.info(f"Backing up existing {OPERATE} directory to {backup_path}")
+        shutil.copytree(self._path, backup_path)
```

Comment:
What if there is aleady a directory with that name ?

Reply:
This should never happen, because as soon as we backup one version we also update the `operate.version` file to the next version. So, the next backup will be a different name.